tests/smoke.runner.js

```diff
@@ -679,6 +679,58 @@ const mochaHooksTestrunner = async () => {
     assert.strictEqual(skippedSpecs, 0)
 }
 
+// ****************************
+// *** Tests for CLI --spec ***
+// ****************************
+const runSpecsWithFlagAllPassed = async () => {
+    const { skippedSpecs } = await launch(
+        'runSpecsWithFlagAllPassed',
+        path.resolve(__dirname, 'tests-cli-spec-arg', 'wdio-with-all-passed.conf.js'),
+        {
+            autoCompileOpts: { autoCompile: false },
+            spec: ['test']
+        }
+    )
+    assert.strictEqual(skippedSpecs, 0)
```

Comment:
I am not sure if this actually verifies that all files were run

Reply:
Yes, I agree. Here it would be good to somehow check the number of specs that were launched to make sure that all files were executed. But so far I have not found such an opportunity

Reply:
afaik it is currently only possible to inspect the spec report as done in some other smoke tests

Reply:
Tests have been updated